sources/packages/web/src/components/aest/students/modals/StudentProfileLegacyMatchesModal.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <v-form ref="linkProfileForm">
+    <modal-dialog-base :showDialog="showDialog" title="Confirm profile link"
```

Comment:
There is something called error summary that is there in many forms but also missed in many forms as well. 
![image](https://github.com/user-attachments/assets/e0bc7361-2d04-468a-8f25-e0a9d71662f5)

Reply:
I am ok to try to enforce it. Changed as below.
![image](https://github.com/user-attachments/assets/f3df0ea9-d41b-4878-a550-0cf3d2e9def8)